diff -ru ezstream-1.0.2/src/stream.c ezstream-1.0.2.mp3-usage-audio-patch/src/stream.c
--- ezstream-1.0.2/src/stream.c	2021-02-11 00:52:27.000000000 +0100
+++ ezstream-1.0.2.mp3-usage-audio-patch/src/stream.c	2021-04-08 11:50:26.480297394 +0200
@@ -225,7 +225,7 @@
 		break;
 	case CFG_STREAM_MP3:
 		if (SHOUTERR_SUCCESS !=
-		    shout_set_content_format(s->shout, SHOUT_FORMAT_MP3, 0, NULL)) {
+		    shout_set_content_format(s->shout, SHOUT_FORMAT_MP3, SHOUT_USAGE_AUDIO, NULL)) {
 			log_error("stream: %s: format: %s: %s",
 			    s->name, cfg_stream_get_format_str(cfg_stream),
 			    shout_get_error(s->shout));
